src/engine/strat_engine/backstore/devices.rs

```diff
@@ -340,6 +484,8 @@ pub fn wipe_blockdevs(blockdevs: &[StratBlockDev]) -> StratisResult<()> {
     let unerased_devnodes: Vec<_> = blockdevs
         .iter()
         .filter_map(|bd| match bd.disown() {
+            // NOTE: We may want to rethink error messaging here because
```

Comment:
Please make ```NOTE``` ```TODO``` instead, so it stands out a little better. Try to reserve ```NOTE``` for significant explanatory matter.

Reply:
Resolved by fixing the TODO.